### PR TITLE
Refactor dev requirements out of local

### PR DESCRIPTION
**Pull request by jgmize**
@jwhitlock r?

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    --hash=sha256:40bfd01002ceaf9e6c5d371d3c4d541d378a4214ea03e45f934980766a809384
+pytest-django==2.9.1 \
+    --hash=sha256:8be15b637738c8cbd1422a6461465c0aeab7839cf76ad2b5d190b6f1f53facd6 \
+    --hash=sha256:743d0056e127ef424850ea76d93d45c92c313da0e56765806a59fc7680c25ab7
+
+# Run tests in a virtual environment
+tox==2.3.1 \
+    --hash=sha256:1823c93ca378092c10bbde428213d3f5066b30adb09e5c001087a83e3e0a402a \
+    --hash=sha256:bf7fcc140863820700d3ccd65b33820ba747b61c5fe4e2b91bb8c64cb21a47ee
+# Enables ./manage.py runserver_plus with in-browser exception debugging
```
Comment (jwhitlock) [inline requirements/dev.txt]: Blank space between ``tox`` and ``Werkzeug`` comment please.

**Comment by jwhitlock** [comment]
TravisCI is happy, and the requirements that moved look good to me.

Add ``dev.txt`` line to ``requirements/README.rst``, and update the Future if you are bold enough.

**Comment by jwhitlock** [comment]
Also, prefix commits with ``bug 1218563:``

**Comment by jgmize** [comment]
Thanks for the quick feedback. I added the missing space, updated requirements/README.rst, and prefixed the commit message with the bug #. I went ahead and amended the commit this time as the changes were only docs and whitespace, but let me know if you'd prefer to always have separate commits and squash only after r+ and I'll stick to that in the future.

**Comment by jwhitlock** [comment]
👍 I prefer amended, rebased commits. I find them easier to read 6+ months later, and GitHub support for them gets better every update.

**Pull request merged**